src/pages/study-room/hook/useTimer.ts

```diff
@@ -0,0 +1,50 @@
+import { useRef, useState } from 'react';
+import useStepStatus from './useStepStatus';
+import { studyStep } from '../util/studyStep';
+import useAudio from './useAudio';
+
+interface UseTimeProps {
+  updateAt: string;
+  step: string;
+}
```

Comment:
요렇게 되어야 하지 않을까요?
```suggestion
interface UseTimerProps {
  updateAt: string;
  step: string;
}
```

Reply:
컴포넌트가 아니기 때문에 꼭 -Props는 적용해주지 않아도 괜찮을 것 같네요!
```suggestion
interface UseTimer {
  updateAt: string;
  step: string;
}
```